src/odemis/gui/cont/tabs.py

```diff
@@ -2825,9 +2833,34 @@ def _onSEMpxs(self, pixel_size):
         best_mpp = self._sem_view.mpp.clip(best_mpp)
         self._sem_view.mpp.value = best_mpp
 
+    def _onAlignPos(self, pos):
+        """
+        Called when the aligner is moved (and the tab is shown)
+        :param pos: (dict str->float or None) updated position of the aligner
+        """
+        # Return if aligner doesn't have both MD_FAV_POS_ACTIVE and MD_FAV_POS_DEACTIVE metadata
+        md = self._aligner_xy.getMetadata()
+        if (model.MD_FAV_POS_ACTIVE and model.MD_FAV_POS_DEACTIVE) not in md:
+            return
+        # Move aligner on tab showing to FAV_POS_ACTIVE position (if all axes are referenced)
+        if not self._aligner_activated:
+            if all(self._aligner_xy.referenced.value.values()):
+                self._aligner_xy.moveAbs(md[model.MD_FAV_POS_ACTIVE]).result()  # Wait till movement done
+                self._aligner_activated = True
+            return
+
+        # Check if updated position is close to FAV_POS_DEACTIVE
+        dist_deactive = math.hypot(pos["x"] - md[model.MD_FAV_POS_DEACTIVE]["x"],
+                                   pos["y"] - md[model.MD_FAV_POS_DEACTIVE]["y"])
+        if dist_deactive <= 0.1e-3:  # within 0.1mm of deactive is considered deactivated.
+            logging.warning("Aligner seems parked, not updating FAV_POS_ACTIVE")
+            return
+        # Save aligner position as the "calibrated" one
+        self._aligner_xy.updateMetadata({model.MD_FAV_POS_ACTIVE: pos})
+
     @classmethod
     def get_display_priority(cls, main_data):
-        if main_data.role == "secom":
+        if main_data.role in ("secom", 'cryo-secom'):
```

Comment:
Please use double-quotes. I usually only have a small preference for them, but here, I'd be quite picky, because the other string already uses that.